src/main/inspect.ml

```diff
@@ -0,0 +1,124 @@
+(*
+Copyright (2018) iNuron BVBA
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*)
+
+open Lwt.Infix
+open Node_cfg
+
+module S = (val (Store.make_store_module (module Batched_store.Local_store)))
+
+let ensure_dir d p =
```

Comment:
mkdir_p as in `mkdir -p` ?

Reply:
I could add a `mkdir_p` function and use it, but it would have different behavior than this function. Would you want that?
This function is only about ignoring the `EEXIST` error, for which I think `ensure_dir` is a good name...
(But indeed part of `mkdir -p` behavior is that it ignores the `EEXIST` error)

Reply:
You are right.